assets/helpers/abTests/helpers/annualContributions.js

```diff
@@ -99,7 +98,7 @@ const VariantA = {
   ],
 };
 
-export const getAnnualAmounts = (annualTestVariant: AnnualContributionsTestVariant) => {
+export const getAnnualAmounts = (annualTestVariant: string) => {
```

Comment:
This is far from ideal, but the tests model doesn't make this easy at the moment (everything is just a string).
We can revisit this later
@joelochlann

Reply:
yeah, what we ultimately need is an `AmountsTest` model and a single function which will get the amounts for all contribution types, based on default amounts overridden by any test participations. but this can wait given current time pressures